spec/rails_helper.rb

```diff
@@ -69,4 +69,19 @@
   config.include Devise::TestHelpers, type: :controller
   config.include SpecHelpers
   config.include Delorean
+
+  if ENV['RSPEC_TASK'] != 'spec:nofeatures'
```

Comment:
I'm going to see if I can do a better job of scoping feature-only config to just the feature specs. But you're right, presently the name is misleading because it gets mixed in for all tests as soon as any feature is required.